addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -237,9 +237,22 @@ def _l10n_it_post_invoices_step_2(self, invoices):
                 else:
                     to_return[invoice] = {'attachment': invoice.l10n_it_edi_attachment_id, 'success': True}
             elif state == 'notificaScarto':
-                errors = [element.find('Descrizione').text for element in response_tree.xpath('//Errore')]
-                to_return[invoice] = {'error': self._format_error_message(_('The invoice has been refused by the Exchange System'), errors), 'blocking_level': 'error'}
-                invoice.l10n_it_edi_transaction = False
+                elements = response_tree.xpath('//Errore')
+                error_codes = [element.find('Codice').text for element in elements]
+                errors = [element.find('Descrizione').text for element in elements]
```

Comment:
@lordkrandel You told me that if we have duplicated invoice, there will be only one error.  There is quite some logic about the index, ...  What do we do?

Reply:
It's just my reasoning, but I didn't find any spec confirming it. Let's keep it.
It's not some specialized "duplicated invoice" error case that will increase the complexity.